src/main/drivers/pwm_output.c

```diff
@@ -246,33 +227,39 @@ void motorDevInit(const motorDevConfig_t *motorConfig, uint16_t idlePulse, uint8
 {
     memset(motors, 0, sizeof(motors));
     
-    uint32_t timerMhzCounter = 0;
     bool useUnsyncedPwm = motorConfig->useUnsyncedPwm;
 
+    unsigned usMin = 0;
```

Comment:
It is a bit strange to use integer and divide it to float a bit later.
With `float`, using base SI units is natural (for me) and it reduces confusion